src/main/java/ev3dev/sensors/ev3/EV3ColorSensor.java

```diff
@@ -56,6 +53,7 @@
     private static final int []lightColor = {Color.NONE, Color.RED, Color.BLUE, Color.WHITE, Color.WHITE, Color.WHITE, Color.WHITE};
     protected short[]raw = new short[3];
 
+    /*
     private void initModes() {

```

Comment:
I think the commented parts are not needed to be in there anymore. If it is for backup, we can luckily use the Git history.